src/classes/Country.cls

```diff
@@ -131,4 +129,52 @@ public with sharing class Country {
         
         System.debug(countryWithCodeMap.get(countrycode));
     }
-}
\ No newline at end of file
+
+    /**   Description :   Method Gettting the Country with letter and Greater District Form Integer
+    *     Created By  :   Sagar Nirwal
+    *     Arguments   :   String,Integer
+    *     Return      :   void
+    *     Revision log:   V_1.0 - Created  - Youe name - Date
+    *                     V_1.1 - Modified - Youe Name - Date - Details what you modified 
+    **/
+    public static void combinationCountryStateDistrict(String combinationLetter,Integer districtSize){
+        
+        Map<String,List<String>> mapOfState = new  Map<String,List<String>>();
+        List<String> lisyOfDistrict = new List<String>();
+		
+        String countryStartLetter = '';
+        for(String countryName : countryMap.keySet()){
+            if(countryName.startsWith(combinationLetter)){
+                mapOfState = countryMap.get(countryName);
+                for(String stateName : mapOfState.keySet()){
```

Comment:
Use countryMap.get(countryName).KeySet() directly no need to create new map and then hold.